src/quo2/components/numbered_keyboard/keyboard_key/view.cljs

```diff
@@ -0,0 +1,34 @@
+(ns quo2.components.numbered-keyboard.keyboard-key.view
+  (:require [react-native.core :as rn]
+            [quo2.theme :as quo.theme]
+            [status-im.ui.components.icons.icons :as icons]
+            [quo2.foundations.colors :as colors]
+            [quo2.components.numbered-keyboard.keyboard-key.style :as style]))
+
+(defn- get-label-color
```

Comment:
I think we should put this helper in the styles file 👍

Reply:
It's not needed anymore. I removed it.

Reply:
Done